cvmfs/download.cc

```diff
@@ -422,8 +422,8 @@ void *DownloadManager::MainDownload(void *data) {
     } else {
       timeout = -1;
       gettimeofday(&timeval_stop, NULL);
-      download_mgr->statistics_->transfer_time +=
-        DiffTimeSeconds(timeval_start, timeval_stop);
+      perf::Xadd(download_mgr->counters_->sz_transfer_time,
+        1000 * DiffTimeSeconds(timeval_start, timeval_stop));
```

Comment:
Now we are counting miliseconds, not seconds. This is because now the counter is not a double, but a long, so in order be more accurate I had to change it